app/Http/Controllers/FishRecordController.php

```diff
@@ -44,4 +44,15 @@ public function index()
             return view('Fish_Record_list',['data'=>$data]);
 
     }
+
+    public function edit(Request $request,$id){
+        $records=Fish_Record::find($id);
```

Comment:
細かいですが次回のプルリクで対応お願いします！
(他のところも)
```suggestion
        $records = Fish_Record::find($id);
```